Makefile

```diff
@@ -86,3 +86,13 @@ clean:
 	find . -type d -name "__pycache__" -delete
 	rm -f .coverage tags
 	rm -rf examples/convert/{main.fmf,test.md,Manual}
+
+requre_install:
+	pip install git+https://github.com/packit/requre.git
```

Comment:
Perhaps, should this go to `setup.py` so that `pip install .[tests]` covers `requre` as well? Btw, is it necessary to install from github? Can we use pypi as we do for all other required modules?

Reply:
yep, could be as part of setup.py, I just wanted new version but now, it is already published to pypi so it is fine to change it there.